magma_access_gateway_configurator/__init__.py

```diff
@@ -2,12 +2,19 @@
 # Copyright 2022 Canonical Ltd.
 # See LICENSE file for licensing details.
 
+import base64
 import logging
 import os
 import sys
 from argparse import ArgumentParser
+from typing import Union
 
+import snowflake  # type: ignore[import]
```

Comment:
Yup, that's the one. 
It used to generate hardware ID which is needed to register AGW with Orc8r.
I'm using it because that's what Magma is using. 
Alternatively, I could run `show_gateway_info.py` delivered by Magma and parse the output to get the ID. I thought the current implementation would be cleaner and more pythonic though.

Reply:
I'd rather not rely a package that is not maintained anymore. Could we use the `uuid` python package?

Reply:
Unfortunately, we can't use any of the solutions you guys proposed, unless we change Magma code. The problem is that the UUID generated by `snowflake` is also used by `magmad` as well (main service of the AGW) to identify the AGW instance. This means that we need to pass the same ID to the Orc8r if we wanna integrate them.
As mentioned above, if we don't want to add another dependency, we can parse the output of the official script which will  run `snowflake` anyway.

Reply:
After discussing with @gruyaume, I've changed that to simply print the output of the official Magma script providing necessary info.
This way we will avoid introducing outdated dependencies as well as potential parsing issues if script's output changes.
How does that sound?